src/__tests__/__snapshots__/links.test.js.snap

```diff
@@ -0,0 +1,39 @@
+// Vitest Snapshot v1
+
+exports[`the build should not break any links 1`] = `
+Set {
```

Comment:
There must be more links than this 🤔
What about all the headings? I know there are some experimental links too.

Reply:
I initially included functionality to detect broken anchor tags but removed it to reduce complexity - I didn't think breaking an anchor tag is as much of an issue. Do redirects work for anchor tags? If we expect developers to add redirects for anchor tags then I can add this back in 😀

Reply:
Yeah I guess as long as you're on the right page it doesn't matter 😄  We are still missing lots of /api links though from the legacy site

Reply:
Do you think we need this test to have confidence in the migration? If so can traverse the legacy site too

Reply:
@AndyClifford I think that would be the main thing we would want it for 👍

Reply:
@AndyClifford Suggestion: Confirm if redirects work for anchor tags with Astro. 

Agree that broken anchors are less of a problem than broken links but they still contribute to a lesser experience. I'd prefer they were included in any checks ❤️

Reply:
I have added support for anchor tags

Reply:
Redirects do not support anchor tags but this is okay - we can validate changes in the snapshot to ensure redirects have been added properly